Makefile

```diff
@@ -27,7 +28,7 @@ prod-up:
 	docker-compose -f ./docker-compose.yaml -f ./docker-compose.nginx-proxy.yaml up -d --build -- server-prod client-prod
 
 beta-up:
-	BETA_PREFIX=$$(git rev-parse --abbrev-ref HEAD) docker-compose -f ./docker-compose.yaml -f ./docker-compose.nginx-proxy.yaml up -d --build --force-recreate -- client-beta server-beta
+	BETA_PREFIX=$$(git rev-parse --short HEAD) $(DOCKER_COMPOSE_BETA) -f ./docker-compose.yaml -f ./docker-compose.nginx-proxy.yaml up -d --build --force-recreate -- client-beta server-beta
```

Comment:
What does these first two options (the commit hash and branch name) do?

Reply:
BETA_PREFIX is used in this part of `docker-compose.yml`, where it defines the host

So in change, it would be hosted in `https://{commit_hash}.dev.mochatto.com`

https://github.com/Amijakan/mochatto/blob/ef5d9ce672aae0f1bf477b9d828a75dd9013f569/docker-compose.nginx-proxy.yaml#L26

`DOCKER_COMPOSE_BETA` is defining the project name, which defines the conainer names.

With changes made by changing CI/CD pipeline from Jenkins to teamcity, the container names collided and was not able to keep the old containers up.

In this image, `412b4a05fbbe42e1` was before the change and `rf-358` is after the change. 

![image](https://github.com/Amijakan/mochatto/assets/9146455/60d0c12a-eab2-43b8-bde8-70f8819c0cd5)

Reply:
Ohh I see thanks sounds good lgtm!